### ISSUE TITLE
Error message: Failed to import block to chain

### ISSUE THREAD

**Issue by cai-dclab** [hyperledger/besu#3395]
### Description
hello

I am struggling with the error of "Failed to import block to chain".
and I found this closed issue: https://github.com/hyperledger/besu/issues/2899

I am not sure if it is the same problem, my network is still running and this error only showed 2 times in the past week.
(running on AWS)

I want to identify if this problem also belongs to issue #2899,
(then I can solve it with upgrade the version),
Or I have to do other research related to verifying the blockhash or something.
(want to know what's meaning of the error message below)

I'm sorry that I couldn't provide more environmental information because I am very fresh in this field.
If any advice that I can get to make a research direction, that will be really helpful.
Thank you in advance.

Software version: version: 

Error message:
```
New chain head detected (block number=222781), currently mining on top of 222780.
Imported #222,781 / 0 tx / 0 om / 0 (0.0%) gas / (0x7a4116a9112ea24762ed30e4c6f65507b921038f82ea17d875adde79863b3388) in 0.000s. Peers: 16
Caching logs bloom for block 0x3663d.
Failed to import block to chain. block=222781 extraData=IbftExtraData[vanityData=0x0000000000000000000000000000000000000000000000000000000000000000, 
seals=[SECP256K1.Signature{r=31067947231597307515973088086785511695430782653315401734983116471929184051551, s=41100377914765765404298290283756800917726635099840850440833293306229461546982, recId=1}, 
SECP256K1.Signature{r=47782506287434789985199491837321874102405210378675089875974341358627722049426, s=56477448400379054562010499330983298925376366053806129530627296595708347561354, recId=0}, 
SECP256K1.Signature{r=31067947231597307515973088086785511695430782653315401734983116471929184051551, s=41100377914765765404298290283756800917726635099840850440833293306229461546982, recId=1}, 
SECP256K1.Signature{r=90592021223121448214823925170077787411866036053198613281180515164735438305430, s=26702380845072626744248573333825194847417154529560575364938790705076047148372, recId=1}], 
vote=Optional.empty, round=0, 
...

Importing block to chain. round=ConsensusRoundIdentifier{Sequence=222781, Round=0}, hash=0x7a4116a9112ea24762ed30e4c6f65507b921038f82ea17d875adde79863b3388
```

**Frequency:** [What percentage of the time does it occur?]

### Versions (Add all that apply)
* Software version: besu/v1.5.3/linux-x86_64/oracle_openjdk-java-14
* Java version: [`java -version`]
* OS Name & Version: [`cat /etc/*release`]
* Kernel Version: [`uname -a`]
* Virtual Machine software & version: [`vmware -v`]
* Docker Version: [`docker version`]
* Cloud VM, type, size: [Amazon Web Services I3-large]

**Comment by macfarla** [comment]
we have fixed a lot of things since besu 1.5.3 - definitely recommend you update!

**Comment by macfarla** [comment]
Do let us know if you're still having issues after updating
